src/BrazeKit-dev.js

```diff
@@ -792,10 +794,13 @@ var constructor = function () {
         braze.addSdkMetadata(['mp']);
         primeBrazeWebPush();
 
-        if (forwarderSettings.userIdentificationType === 'MPID' && mParticle.Identity != null && mParticle.Identity.getCurrentUser().getMPID() != null) {
```

Comment:
If we're no longer referencing `forwarderSettings`, we should remove it from the function parameters.

Reply:
It is still being passed to `openSession` at the bottom of this function

Reply:
Ooof. Missed that.